### PR TITLE
2.2.0 release for G/R

### PR DESCRIPTION
**Pull request by tylerjw**
This is to release the latest changes into galactic and rolling.

Please use the rebase option on this.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Changelog for package moveit_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.2.0 (2021-12-10)
+------------------
+* Add hybrid planning action definitons (`#135 <https://github.com/ros-planning/moveit_msgs/issues/135>`_)
```
Comment (AndyZe) [inline CHANGELOG.rst:7]: It's strange that this URL doesn't work. I guess it was auto-generated? It's a pull request, not an issue

Should be

https://github.com/ros-planning/moveit_msgs/pull/135

Comment (tylerjw) [reply CHANGELOG.rst:7]: I think github must have changed :(

These generated links used to work.

Comment (tylerjw) [reply CHANGELOG.rst:7]: Here is the code that generates this: https://github.com/ros-infrastructure/catkin_pkg/blob/e5f553826cff83b18cf2d438c7df6c13b3c95c4a/src/catkin_pkg/changelog_generator_vcs.py#L261

All the other links are also broken in this changelog.  /issues/ used to point to pull requests, github must have changed.

Comment (tylerjw) [reply CHANGELOG.rst:7]: Created this issue: https://github.com/ros-infrastructure/catkin_pkg/issues/319

Comment (AndyZe) [reply CHANGELOG.rst:7]: Shall we merge it anyway? It's no worse than any of the rest of the Changelogs (and really, it's not a huge issue anyway).

**Review by AndyZe — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (tylerjw) [reply CHANGELOG.rst:7]: I think so... I don't think we should block our releases on this issue.  No harm in just fixing this later once the ros-infrastructure catches up.

Comment (vatanaksoytezer) [reply CHANGELOG.rst:7]: It works for me, it actually redirects to pull link.

Comment (tylerjw) [reply CHANGELOG.rst:7]: It now seems to be fixed.  Maybe github itself was broken?

**Pull request merged**